bundles/statistics/statsgrid2016/components/ExtraFeatures.js

```diff
@@ -1,26 +1,94 @@
-Oskari.clazz.define('Oskari.statistics.statsgrid.ExtraFeatures', function(sandbox, locale) {
+Oskari.clazz.define('Oskari.statistics.statsgrid.ExtraFeatures', function(sandbox, locale, instance) {
```

Comment:
I can't find a reference to instance on the code that was added. Why is it passed here?